include/vulkan/utility/vk_format_utils.h

```diff
@@ -0,0 +1,2297 @@
+// *** THIS FILE IS GENERATED - DO NOT EDIT ***
+// See format_utils_generator.py for modifications
+// Copyright 2023 The Khronos Group Inc.
+// Copyright 2023 Valve Corporation
+// Copyright 2023 LunarG, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <vulkan/vulkan.h>
+
+#include <stdbool.h>
+
+#define FORMAT_UTILS_MAX_PLANES 3
+#define FORMAT_UTILS_MAX_COMPONENTS 4
+
+enum FORMAT_UTILS_NUMERICAL_TYPE {
+    FORMAT_UTILS_NUMERICAL_TYPE_NONE = 0,
+    FORMAT_UTILS_NUMERICAL_TYPE_SFLOAT,
+    FORMAT_UTILS_NUMERICAL_TYPE_SINT,
+    FORMAT_UTILS_NUMERICAL_TYPE_SNORM,
+    FORMAT_UTILS_NUMERICAL_TYPE_SRGB,
+    FORMAT_UTILS_NUMERICAL_TYPE_SSCALED,
+    FORMAT_UTILS_NUMERICAL_TYPE_UFLOAT,
+    FORMAT_UTILS_NUMERICAL_TYPE_UINT,
+    FORMAT_UTILS_NUMERICAL_TYPE_UNORM,
+    FORMAT_UTILS_NUMERICAL_TYPE_USCALED
```

Comment:
nit: suggestion to end last enum line with a comma, to remove corresponding line from the future diffs if new members are added.